Examples/PurchaseTester/Constants.swift

```diff
@@ -0,0 +1,21 @@
+//
```

Comment:
Could we have this live in `Tests/TestingApps` instead since it's not really an example?

Reply:
I think this makes sense! I can move it 💪